test/spec/modules/userId_spec.js

```diff
@@ -14,21 +14,23 @@ import {unifiedIdSubmodule} from 'modules/userId/unifiedIdSystem';
 import {pubCommonIdSubmodule} from 'modules/userId/pubCommonIdSystem';
 import {id5IdSubmodule} from 'modules/id5IdSystem';
 import {identityLinkSubmodule} from 'modules/identityLinkIdSystem';
+import {parrableIdSubmodule} from 'modules/parrableIdSystem';
```

Comment:
Can you add your unit tests in separate spec file ? I will open an issue to do the same for other id systems as well.

Reply:
yes sure. will work on it.